api_client/python/timesketch_api_client/client.py

```diff
@@ -414,21 +415,41 @@ def get_aggregator_info(self, name='', as_pandas=False):
 
         return pandas.DataFrame(lines)
 
-    def list_sketches(self):
+    def list_sketches(self, per_page=50, scope='user', include_archived=True):
         """Get a list of all open sketches that the user has access to.
 
-        Returns:
-            List of Sketch objects instances.
+        Args:
+            per_page: Number of items per page when paginating. Default is 50.
+            scope: What scope to get sketches as (e.g user, admin, shared)
```

Comment:
can you add the default avalue and add that this is optional to all three items here?